Emby.Server.Implementations/Plugins/PluginManager.cs

```diff
@@ -374,7 +418,7 @@ private bool ChangePluginState(LocalPlugin plugin, PluginStatus state)
         private LocalPlugin? GetPluginByAssembly(Assembly assembly)
         {
             // Find which plugin it is by the path.
-            return _plugins.FirstOrDefault(p => string.Equals(p.Path, Path.GetDirectoryName(assembly.Location), StringComparison.Ordinal));
+            return _plugins.FirstOrDefault(p => p.DllFiles.Contains(assembly.Location, StringComparer.Ordinal)));
```

Comment:
```suggestion
            return _plugins.FirstOrDefault(p => p.DllFiles.Contains(assembly.Location, StringComparer.Ordinal));
```

Accidentally suggested extra `)`